src/meson.build

```diff
@@ -0,0 +1,33 @@
+modbus_version_h = configure_file(input : 'modbus-version.h.in',
+                                  output : 'modbus-version.h',
+                                  configuration : ver_conf)
+
+sources = files([
+    'modbus.c',
+    'modbus-data.c',
+    'modbus-rtu.c',
+    'modbus-tcp.c',
+])
+
+headers = [
+    'modbus.h',
+    modbus_version_h,
+    'modbus-rtu.h',
+    'modbus-tcp.h',
+]
+install_headers(headers, subdir : 'modbus')
+
+lt_current = 6
+lt_revision = 0
+lt_age = 1
+lt_version = '@0@.@1@.@2@'.format(lt_current, lt_age, lt_revision)
```

Comment:
libtool version actually maps a bit funky to regular versions, the math for it is `(C - A).(A).(R)`.

Reply:
Interesting. I changed it to:
```
lt_current = 6
lt_revision = 0
lt_age = 1
lt_version = '@0@.@1@.@2@'.format(lt_current - lt_age, lt_age, lt_revision)
```
and I get the following files created:
```
lrwxrwxrwx 1 jringle domain users     14 Nov  8 23:58  libmodbus.so -> libmodbus.so.6*
-rwxr-xr-x 1 jringle domain users 121176 Nov  8 23:58  libmodbus.so.5.1.0*
lrwxrwxrwx 1 jringle domain users     18 Nov  8 23:58  libmodbus.so.6 -> libmodbus.so.5.1.0*
```
Is this correct?

Reply:
Well, you need one more small tweak I just mentioned below.